packages/node/src/transports/base.ts

```diff
@@ -55,17 +57,15 @@ export abstract class BaseTransport implements Transport {
   }
 
   /** Returns a build request option object used by request */
-  protected _getRequestOptions(address: string | url.URL): http.RequestOptions | https.RequestOptions {
-    if (!isString(address) || !(address instanceof url.URL)) {
-      throw new SentryError(`Incorrect transport url: ${address}`);
-    }
-
+  protected _getRequestOptions(url: url.URL): http.RequestOptions | https.RequestOptions {
     const headers = {
-      ...this._api.getRequestHeaders(SDK_NAME, SDK_VERSION),
+      // The auth headers are not included because auth is done via query string to match @sentry/browser.
+      //...this._api.getRequestHeaders(SDK_NAME, SDK_VERSION)
       ...this.options.headers,
     };
-    const addr = address instanceof url.URL ? address : new url.URL(address);
-    const { hostname, pathname: path, port, protocol } = addr;
+    const { hostname, port, protocol } = url;
+    // See https://github.com/nodejs/node/blob/38146e717fed2fabe3aacb6540d839475e0ce1c6/lib/internal/url.js#L1268-L1290
+    const path = `${url.pathname || ''}${url.search || ''}`;
```

Comment:
The helper in node includes the query string in the path, and we should too -- otherwise we'd miss the auth info.